db/migrate/20211126183839_create_product.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProduct < ActiveRecord::Migration[6.1]
+  def change
+    create_table :products do |t|
+      t.string :name
+      t.string :category
+      t.decimal :price, precision: 6, scale: 2
```

Comment:
let's store price in integer and in cents. 
so if product price is 100rs, we store price_in_cents -> 10000 and create a method called price that returns 100

Reply:
Ok Sir

Reply:
price_in_cents used.
price method created.